persistence/sql/identity/persister_identity.go

```diff
@@ -651,6 +643,12 @@ func (p *IdentityPersister) CreateIdentities(ctx context.Context, identities ...
 	}); err != nil {
 		return err
 	}
+
+	// Report succeeded identities as created.
+	for _, identID := range succeededIDs {
```

Comment:
I'm not entirely sure if this is entirely correct or if we can end up with an error in the transaction block but still have the identity partially available.

Reply:
No, I think this is really the right place.